.github/workflows/ansible-test.yml

```
@@ -185,71 +195,9 @@ jobs:
           - '3.10'
           - '3.11'
         include:
-          # Add more CI runs for specific ansible-core + Python combinations
-
-          # The commented branches below are EOL,
-          # do you really need your collection to support them if it still does?
-
-          # Ansible 2.9 - only if your collection supports Ansible 2.9!
-          # - ansible: stable-2.9
-          #   python: '2.6'
-          # - ansible: stable-2.9
-          #   python: '2.7'
-          # - ansible: stable-2.9
-          #   python: '3.5'
-          # - ansible: stable-2.9
-          #   python: '3.6'
-          # - ansible: stable-2.9
-          #   python: '3.7'
-          # - ansible: stable-2.9
-          #   python: '3.8'
-          # ansible-base 2.10 - only if your collection supports ansible 2.10!
-          # - ansible: stable-2.10
-          #   python: '2.6'
-          # - ansible: stable-2.10
-          #   python: '2.7'
-          # - ansible: stable-2.10
-          #   python: '3.5'
-          # - ansible: stable-2.10
-          #   python: '3.6'
-          # - ansible: stable-2.10
-          #   python: '3.7'
-          # - ansible: stable-2.10
-          #   python: '3.8'
-          # - ansible: stable-2.10
-          #   python: '3.9'
-          # ansible-core 2.11 - only if your collection supports ansible 2.11!
-          # - ansible: stable-2.11
-          #   python: '2.6'
-          # - ansible: stable-2.11
-          #   python: '2.7'
-          # - ansible: stable-2.11
-          #   python: '3.5'
-          # - ansible: stable-2.11
-          #   python: '3.6'
-          # - ansible: stable-2.11
-          #   python: '3.7'
-          # - ansible: stable-2.11
-          #   python: '3.8'
-          # - ansible: stable-2.11
-          #   python: '3.9'
-          # ansible-core 2.12 - only if your collection supports ansible 2.12!
-          # - ansible: stable-2.12
-          #   python: '2.6'
-          # - ansible: stable-2.12
-          #   python: '2.7'
-          # - ansible: stable-2.12
-          #   python: '3.5'
-          # - ansible: stable-2.12
-          #   python: '3.6'
-          # - ansible: stable-2.12
-          #   python: '3.7'
-          # - ansible: stable-2.12
-          #   python: '3.8'
-          # - ansible: stable-2.12
-          #   python: '3.9'
-          # - ansible: stable-2.12
-          #   python: '3.10'
+          # Timely add new versions announced in
+          # https://github.com/ansible-collections/news-for-maintainers,
+          # consider dropping testing against EOL versions.
```

```suggestion
          # consider dropping testing against EOL versions and versions you don't officially support.
```

Similar to my comment above.